src/main/resources/assets/dynamiccrosshair/lang/zh_cn.json

```diff
@@ -33,6 +40,7 @@
   "text.dynamiccrosshair.option.mode.Always": "总是",
   "text.dynamiccrosshair.option.mode.IfTargeting": "若瞄准",
   "text.dynamiccrosshair.option.mode.IfInteractable": "若可交互",
+  "text.dynamiccrosshair.option.mode.IfFullyDrawn": "If Fully Drawn", // It's for ranged weapons, to change the crosshair style if the weapon is fully drawn (bow or trident), or has an arrow/firework loaded (crossbow)
```

Comment:
For future reference, JSON does not support comments, frustratingly. So this will fail to load the Chinese translation at all, because the whole file will be a malformed JSON.
For this case, I'll merge this and then remove the comment & rename the key to something more understandable for translators in a follow-up.

Reply:
Uh oh